README.md

```diff
@@ -1,4 +1,4 @@
-# ![logo](./docs/figures/logo.svg "logo")
+# ![logo](./docs/figures/logo.png "logo")
```

Comment:
Update this link to fix broken image in PyPI.

https://stackoverflow.com/a/46875147/10241324

![image](https://github.com/vprusso/toqito/assets/66048318/a623a2a9-df06-4d7d-97aa-f5ac5b77e3a6)
https://pypi.org/project/toqito/